### ISSUE TITLE
FilterBar: missing accessibility properties for filterGroupItem

### ISSUE THREAD

**Issue by nidhi0104** [SAP/ui5-webcomponents-react#3573]
**Describe the bug**
Screen renders don't read "required" for the fields that are enabled required.

**Isolated Example**
UI Link
Refer: https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--default-story

**To Reproduce**
1. Set property 'required' of filterGroupItem as true.
2. Asterik(*) will come for the first filterGroupItem
3. The screen render don't read out the 'required' word 

**Expected behavior**
It should read out 'required' if some field is set as required

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:1.6.0
`@ui5/webcomponents-react` version:0.27.5
Operating System:macOs
Browser:chrome

**Comment by Lukas742** [comment]
Hi @nidhi0104 

we're currently blocked to implement this on our end, as not all ui5 web components do yet support the `required` attribute. I've created issues regarding this in their repository (e.g. https://github.com/SAP/ui5-webcomponents/issues/5895) 
Until then you can just add `required` to the input you pass as child to the `FilterGroupItem` (if it's supported).

E.g.:
```jsx
<FilterGroupItem required label="Filter 1">
  <Input required />
</FilterGroupItem>
```

**Comment by Lukas742** [comment]
Hi @nidhi0104 

we discussed this issue internally again and decided against implicitly coupling the `required` prop of the `FilterGroupItem` with the `required` prop of input components passed as `children`.  
The main reason is that we want to avoid cloning the React element (using `cloneElement`). Additionally, keeping the logic decoupled makes the component more flexible, allowing it to support custom inputs that might not implement a `required` prop.
